lib/procfile.js

```diff
@@ -50,7 +50,8 @@ function readProcFile (pid, options, done) {
     history.set(stat, options.maxage)
 
     return done(null, {
-      cpu: cpu,
+      cpu: Math.min(cpu, 100.0),
```

Comment:
This value is meaningless :(
https://github.com/Unitech/pm2/issues/3708#issuecomment-396605583

Reply:
It is meaningless to us because we now that the value can be greater then 100 in theory. Though, it is meaningful for people who just want a value between `0` and `100`. We all have different point of views and this is why I added the `usage` value that reflects reality.

Reply:
>  can be greater then 100 in theory. 

It's not theory, it just "the common case". Actually it should be really easy to get the value above 100%.